src/style.css

```
@@ -1,3 +1,83 @@
+
 body {
   margin: 0;
+padding: 0;
+background-color: #fff;
+border: 0;
+font-family: Roboto, sans-serif;
 }
+
+header {
+display: flex;
+justify-content: space-between;
+align-items: center;
+padding: 0 50px;
+}
```

make correct indent all over the page
```suggestion
header {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 50px;
}
```